app/controllers/dashboard/form/publish_controller.rb

```diff
@@ -26,12 +26,33 @@ def update
         # against the draft validations, then mark the record as published and
         # save again--this time using the published validations. That way the
         # appropriate error messages will appear on the form when it's re-rendered
-        if publish?
+        if publish? && allow_publish?
           # WorkVersion#set_thumbnail_selection may be unreliable if the Shrine::ThumbnailJob is delayed
           @resource.set_thumbnail_selection
           @resource.indexing_source = Proc.new { nil }
           @resource.save
           @resource.publish
+        elsif request_curation?
+          # We want validation errors to block curation requests and keep users on the edit page
+          # so WorkVersion needs to temporarily act like it's being published. It's returned to it's
+          # initial state before being saved.
+          begin
+            @resource.save
+            initial_state = @resource.aasm_state
+            @resource.publish
+            if @resource.valid?
+              CurationTaskExporter.call(@resource.id)
```

Comment:
I believe Airtable actually does handle this. It doesn't allow duplicate IDs (in this case the scholarsphere uuid).

Reply:
Will attempting to create a duplicate ID raise an error in Airrecord? If so, then we may want to bubble that error up, rescue it, and show a message to the user so that a double-click on the button doesn't just result in a 500 error page.

Reply:
Ok, in that case we should probably try to do something to ensure idempotence here. One thing that would probably work would be to do
```ruby
@resource.update_column(:draft_curation_requested, true)
```
right after
```ruby
elsif request_curation?
```
and then first thing in both the `rescue` block and in the `else` condition:
```ruby
@resource.update_column(:draft_curation_requested, false)
```

Reply:
Oh, for that ☝️ to work, we'd also need to make the `elsif` condition
```ruby
elsif request_curation? && allow_curation?
```
which we should do anyway.

Reply:
I fixed a mistake in my last comment ☝️